# Question

Will the code work? Why or why not?

```js
const getUsers = async () => {
  const url = 'https://jsonplaceholder.typicode.com/users';
  const users = await fetch(url).then(r => r.json()).catch((err) => []);
  console.log(users)
  return users;
}
getUsers()
```

Again, running the code gives you the answer, so try to figure it out on your own first!

# Answer
The code works because it awaits the fetch call and converts the response into `json` data, and has a catch handler for errors. 
# Teacher Feedback
Score: 3/3
Solid!